CONTRIBUTING.md

```diff
@@ -121,6 +121,43 @@ cabal test ouroboros-consensus:test:consensus-test --test-show-details=direct
 Note the second one cannot be used when we want to provide CLI arguments to the
 test-suite.
 
+# Generating documentation and setting up hoogle
+
+Documentation contains some tikz figures that require some preprocessing for
+them to be displayed. In order to do this, one can just invoke the `make` rules:
+
+```bash
```

Comment:
I wonder if this `Makefile` should go inside the `docs` folder. People might get confused and think they can build/test the code using it.

Reply:
The makefile is gone now